packages/suite/src/components/suite/modals/ReduxModal/ConfirmValueModal/ConfirmValueModal.tsx

```diff
@@ -90,7 +88,7 @@ export const ConfirmValueModal = ({
     );
 
     const canConfirmOnDevice = !!(device?.connected && device?.available);
-    const isCancelable = isActionAbortable || isConfirmed;
+    const isCancelable = isConfirmed;
```

Comment:
Shouldn't this always be true now?

Reply:
`isActionAbortable` was false only for the old bridge. Since it was used in an "or" condition, it should always be true now, right?

Reply:
I think `isConfirmed` is true after connect call completes, without it you wouldn't be able to close the modal if you were on old bridge